test/Tool/signbits1-invalid.opt

```diff
@@ -0,0 +1,11 @@
+; REQUIRES: solver
+
+; RUN: %souper-check %solver -print-counterexample=false  %s > %t 2>&1
+; RUN: FileCheck %s < %t
+
+; CHECK: Invalid
+%0:i32 = var
+%1:i32 = var
```

Comment:
but the parser accepts s=1, right? this will make it clear to people that this is the negative test case corresponding to the s=2 case, which is positive